src/client/logging/_global.ts

```diff
@@ -70,7 +71,20 @@ function initialize() {
     }
 }
 
-// Register the output channel transport the logger will log into
+// Set the logging level the extension logs at.
+export function setLoggingLevel(level: LogLevel | undefined) {
+    if (level) {
+        const levelName = resolveLevelName(level, winston.config.npm.levels);
+        if (levelName) {
+            globalLogger.level = levelName;
+        }
+    } else {
+        // Disable logging.
```

Comment:
It would worth having a comment here noting other possible behaviors:

```suggestion
        // For now we disable all logging.  One alternative would be
        // to only disable logging to the output channel (by removing
        // the transport from the logger).
```